src/main/java/org/launchcode/liftoffrecipetracker/models/Category.java

```diff
@@ -0,0 +1,59 @@
+package org.launchcode.liftoffrecipetracker.models;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+
+@Entity
+public class Category  {
+
+    //Class Variables
+    @GeneratedValue
+    @Id
+    private int id;
+    // do we want to specify size for category names?
```

Comment:
I think validation would definitely be helpful here! @Size is a good start and I think @NotBlank is also a good choice. Maybe @NotNull as well?

Reply:
done!